### PR TITLE
added tabs to the gui and a display for all of the nodes that are bei…

### PR DESCRIPTION
**Pull request by TrevorDecker**
…ng used

### CODE REVIEW

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/NodeViewer.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package com.roboclub.robobuggy.ui;
+
+import java.util.List;
+import java.util.Random;
+
+import javax.swing.JButton;
+
+import com.roboclub.robobuggy.main.Robot;
+import com.roboclub.robobuggy.ros.Node;
+import com.roboclub.robobuggy.ros.NodeChannel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/NodeViewer.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ros.NodeChannel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1939509489/issues/source?bid=2938665&fileBranchId=3104187#l10)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/RobobuggyGUITabs.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import javax.swing.JTabbedPane;
+
+/**
+ * 
+ * @author Trevor Decker
+ * 
+ * is a robobuggy gui element that contains many other gui elements in seperate tabs
+ *
+ */
+public class RobobuggyGUITabs extends JTabbedPane {
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/RobobuggyGUITabs.java]: Codacy Issue found: [Each class should declare at least one constructor](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1939509492/issues/source?bid=2938665&fileBranchId=3104187#l13)

**Comment by TrevorDecker** [comment]
@VivaanBahl can you take a look at this

**Comment by VivaanBahl** [comment]
I approve of this - tested on both the Surface and my own computer

**Comment by VivaanBahl** [comment]
I'm putting up one more commit though, to get rid of the random placement of the buttons and put them into a nice list

**Pull request merged**